client/extensions/woocommerce/state/ui/orders/selectors.js

```diff
@@ -73,6 +74,8 @@ export const getDefaultEmptyOrder = () => {
 		prices_include_tax: false,
 		billing: {},
 		shipping: {},
+		payment_method: 'calypso_manual',
+		payment_method_title: translate( 'Manual Payment' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Manage Payment' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).